packages/frontend/src/features/comments/hooks/useComments.ts

```diff
@@ -47,7 +49,13 @@ export const useCreateComment = () => {
     >((data) => createDashboardTileComment(data), {
         mutationKey: ['create-comment'],
         onSuccess: async (_, { dashboardUuid }) => {
-            await queryClient.invalidateQueries(['comments', dashboardUuid]);
+            await Promise.all([
```

Comment:
We're adding an additional change to the query client because of our dual state. Sometimes we're using the route params (which might be `slug`), and sometimes we defer to a standard identifier (`dashboard.uuid`). We need to handle both in the hook for the UI to properly update after a successful mutation.